src/build.ts

```diff
@@ -276,7 +291,8 @@ export async function build(
       const outputPath = join(dirname(sourceFile), basename(sourceFile, ".md") + ".html");
       const path = join("/", dirname(sourceFile), basename(sourceFile, ".md"));
       const resolveOutput = (name: string) => join(config.output, resolvePath(path, name));
-      const pageSize = (await stat(join(config.output, outputPath))).size;
+      const pageSize = (await maybeStat(join(config.output, outputPath)))?.size;
+      if (!pageSize) continue;
```

Comment:
Why is this change necessary?

Reply:
In a previous version of the tests in this PR, I wasn't writing the files to disk (because it wasn't needed). Since it was using `effect` methods to write the files, but direct filesystem access here, there was a conflict that caused the tests to fail.

I ended up not going down that route though, so I can revert this.